### ISSUE TITLE
How to parse single testResult?

### ISSUE THREAD

**Issue by dracorp** [jenkinsci/junit-plugin#316]
I'd like to parse single testResult from junit tests and send simple summary via instant messenger.
Is there any way to do this directly from Jenkinsfile or via Jenkins' REST API? For example:
> Tests execution failed (2/89 tests failed)
I don't any idea how to get into results.

**Comment by timja** [comment]
if you're using the pipeline step the return fail will give you passed failed and skipped tests I believe.

Won't give you any more info than that thouggh

**Comment by dracorp** [comment]
@timja Do you mean that junit returns a value which I can analyze?

**Comment by timja** [comment]
yes, example: https://github.com/jenkinsci/junit-plugin/blob/master/src/test/java/io/jenkins/plugins/junit/storage/TestResultStorageJunitTest.java#L129-L130

**Comment by dracorp** [comment]
Great, thank you.
